src/main/scala/ai/privado/languageEngine/csharp/processor/CSharpProcessor.scala

```diff
@@ -123,11 +114,12 @@ class CSharpProcessor(
       .withDownloadDependencies(Try(!config.skipDownloadDependencies).getOrElse(false))
 
     val xtocpg = new CSharpSrc2Cpg().createCpg(cpgconfig).map { cpg =>
-      println(
-        s"${TimeMetric.getNewTime()} - Base processing done in \t\t\t\t- ${TimeMetric.setNewTimeToLastAndGetTimeDiff()}"
-      )
+      statsRecorder.endLastStage()
+      statsRecorder.initiateNewStage("Default overlays")
       // Apply default overlays
       X2Cpg.applyDefaultOverlays(cpg)
+      statsRecorder.endLastStage()
+      statsRecorder.setSupressSubstagesFlag(false)
```

Comment:
In continuation with this comment - https://github.com/Privado-Inc/privado-core/pull/1073/files#r1644326841

Here we are enabling again to print the passes stats (start and end event) to be printed again. As from this point onwards all the passes are of privado.

I don't see these statements for other language processors, can you check and add them?